### ISSUE TITLE
footer, Community - add github repo links

### ISSUE THREAD

**Issue by gerbrent** [JupiterBroadcasting/jupiterbroadcasting.com#224]
We should add the https://github.com/JupiterBroadcasting/jupiterbroadcasting.com repo link:

* [x] in the footer,
* [x] also in the Community page.
* [x] consider linking to other JB issue trackers - i.e.  jupiter.tube and Matrix

**Comment by reclaimingmytime** [comment]
We might also want to link the other JB issue trackers, like for jupiter.tube and Matrix:

https://github.com/JupiterBroadcasting/jupiter.tube/issues
https://github.com/JupiterBroadcasting/matrix/issues

**Comment by CDEguia** [comment]
I second this one, I heard the new Office Hours 9 and went to https://jupiterbroadcasting.net/ to look for the link to the github page, sadly I couldn't find one.

**Comment by gerbrent** [comment]
K, will add it right away

**Comment by gerbrent** [comment]
closed by #227 227

**Comment by elreydetoda** [comment]
Another option we should consider (maybe this is a different ticket, because of the scope of this task) is an "Edit this page" button. That'd let people easily know they can edit and update pages right from their browser.

This'll depend on #29 on if we want to point them to editing the page on GH for example: https://www.vagrantup.com/docs

![Screenshot_20220812-123729_DuckDuckGo.jpg](https://user-images.githubusercontent.com/10230166/184404330-535bf5cc-a90b-4f30-9ec5-5398d1710b8a.jpg)

 or via a CMS for example: https://www.netlifycms.org/docs/intro/

![Screenshot_20220812-123809_DuckDuckGo.jpg](https://user-images.githubusercontent.com/10230166/184404390-9f32b8c4-4e4e-476e-a64e-3951c0b86172.jpg)

**Comment by mrbracchi** [comment]
I want to help and this was marked as a good place to start. Is it on hold until the edit this page thing gets resolved?

**Comment by gerbrent** [comment]
hi @mrbracchi ! thanks for getting involved - super amazing.

My fault for not clearing the "good first issue" tag since this was partly solved... and indeed this secondary goal is a bit more involved and less beginner friendly. Sorry!

I am, though, happy to point you elsewhere if you connect w me directly (via Matrix perhaps if you're there?)

**Comment by StefanS-O** [comment]
I added the links to Github, Matrix and jupiter.tupe issue tracker and also integrated an Edit this page button which takes you to the right file in the repo. @gerbrent i think for a first implementation it is ok:

![Bildschirmfoto 2022-08-22 um 16 07 13](https://user-images.githubusercontent.com/1253938/185941235-fc942b19-83c5-4efe-88f7-11bc311f3731.png)
